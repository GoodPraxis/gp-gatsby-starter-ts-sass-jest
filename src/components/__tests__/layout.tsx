import renderer from 'react-test-renderer';
import React from 'react';
import Layout from '../layout';

describe('Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Layout><div>Foobar</div></Layout>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
