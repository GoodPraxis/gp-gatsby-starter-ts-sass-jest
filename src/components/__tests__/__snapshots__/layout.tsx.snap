// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<main>
  <div>
    Foobar
  </div>
</main>
`;
